.gitignore

```diff
@@ -6,3 +6,6 @@ bazel-out
 bazel-dependencies
 bazel-testlogs
 tool/bazelcache/.terraform/*
+tool/elk/.terraform/*
+tool/elk/*.tfvars
```

Comment:
`.tfvars` file stores the actual values of the secrets and therefore must not get into repository